.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,8 +11,23 @@ Closes: # (issue)
 # Checklist:
 
 - [ ] I have performed a self-review of my own code
+- [ ] I have added a short summary of my change to the CHANGELOG in website/CHANGELOG.md
 - [ ] My code follows the style guidelines of this project
 - [ ] I have commented my code in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
+
+
+# A note about the CHANGELOG
+
+Hello 👋 and thank you for contributing to Gradio!
+
+All pull requests must update the change log located in website/CHANGELOG.md unless they are labeled with the no-changelog-update label.
```

Comment:
Is `website/CHANGELOG.md` the correct path? I don't see this file

Reply:
no you're right, @freddyaboulton this should point to the releases dir now. good catch!